modules/ossm-rn-new-features.adoc

```diff
@@ -134,8 +134,19 @@ This release marks the end of support for {SMProductShortName} Control Planes ba
 * Gateway injection is not a supported feature.
 
 === Kubernetes Gateway API
-Kubernetes Gateway API is a technology preview feature that is disabled by default.
+Kubernetes Gateway API is a technology preview feature that is disabled by default. If the Kubernetes API deployment controller is disabled, you must manually deploy and link an ingress gateway to the created Gateway object.
```

Comment:
```suggestion
:FeatureName: Kubernetes Gateway API
include::snippets/technology-preview.adoc[]

Kubernetes Gateway API is a technology preview feature that is disabled by default. If the Kubernetes API deployment controller is disabled, you must manually deploy and link an ingress gateway to the created Gateway object.
```
I think we need to add in the tech preview snippet here. What do you think? https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features

Reply:
PR updated @bburt-rh